ext/pgsql/pgsql.c

```diff
@@ -983,13 +982,13 @@ PHP_FUNCTION(pg_query)
 	ExecStatusType status;
 
 	if (argc == 1) {
-		if (zend_parse_parameters(ZEND_NUM_ARGS(), "s", &query, &query_len) == FAILURE) {
+		if (zend_parse_parameters(argc, "s", &query, &query_len) == FAILURE) {
```

Comment:
If argc is not used anywhere else, please inline it into the zpp calls.

Reply:
Hmm, I forced pushed a change a few minutes ago where I reviewed all the usages of `argc` in this file, and either inlined it, or used it if `ZEND_NUM_ARGS()` was needed multiple times.

Reply:
Yeah, just ignore me :)